docs/reference/inference/inference-apis.asciidoc

```diff
@@ -52,6 +51,65 @@ For these models, the minimum number of allocations is `0`.
 If there is no {infer} activity that uses the endpoint, the number of allocations will scale down to `0` automatically after 15 minutes.
 
 
+[discrete]
+[[infer-chunking-config]]
+=== Configuring chunking
```

Comment:
Should we add to this description that chunking only occurs when ingesting documents into semantic text fields? To avoid any confusion that the chunking strategy may be run in other inference use cases.